2D Simulation/spec/movesTrackerSpec.js

```diff
@@ -0,0 +1,395 @@
+describe("movesTracker", () => {
+    let imported = require('../movesTracker.js');
```

Comment:
I thought `require` was a RequireJS thing? I haven't done much with JavaScript modules.

Reply:
Surprisingly no! This is actually Node's `require`. RequireJS has much different formatting (it's more akin to dependency injection than an import statement).